scripts/create_x509_chain_pipeline.py

```diff
@@ -0,0 +1,436 @@
+import os
+import re
+import base64
+import logging
+import shutil
+import subprocess
```

Comment:
Do we really need two almost-the-same-but-not-quite-the-same tools -- create_x509_chain.py and create_x509_chain_pipeilne.py?  The distinction between the two is not obvious and it seems like a maintenance nightmare since any changes to one script may or may not apply to the other script.  If create_x509_chain.py didn't have enough functionality to create the certs that the pipeline uses, then maybe you should have added functionality to that tool instead of copying it to make a second tool?  (I remember you telling me why you did this, but the actual reason has slipped my mind by now.)

Reply:
no we don't need both..if you see my comments on the PR description...Please see the first point...I am going to take that as a task for later

---
In reply to: [315400964](https://github.com/Azure/azure-iot-sdk-python-preview/pull/167#discussion_r315400964) [](ancestors = 315400964)